<!DOCTYPE HTML>
<html>
<head>
	<!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-108830011-1"></script>
	<script>
		window.dataLayer = window.dataLayer || [];
		function gtag(){dataLayer.push(arguments);}
		gtag('js', new Date());

		gtag('config', 'UA-108830011-1');
	</script>
	<title>Overview - JSbond</title>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
	<link rel="shortcut icon" href="images/logo_icon.png">
	<link rel="stylesheet" href="assets/css/main.css">
	<link rel="stylesheet" href="assets/css/magnific-popup.css">
	<link rel="stylesheet" href="assets/css/dms.css">
</head>
<body>
	<div id="wrapper">
		<div id="main">
			<div class="inner">
				<header id="header">
					<a class="logo"><strong>Overview</strong> - JSbond</a>
				</header>

					<!-- Content -->
					<section>
						<header class="main">
							<h1>HuR Introduction</h1>
						</header>

						<hr class="major">
						<h2>(1) HuR introduction</h2>
						<p>Human antigen R (hereinafter referred to as HuR), which is one of the RNA-binding proteins (RBPs), is responsible for regulating the splicing, stability and translation of mRNA [1, 2].</p>
						<p>The three characteristics of the HuR protein are as follows:</p>
						<dl>
							<dt>1. It interacts with mRNA.</dt>
							<dd>
								<p>The binding site of the HuR protein to mRNA is predominantly in the adenylate-uridylate- or uridylate-rich sequence elements (ARE) [1] in the 3'-untranslated region (3'-UTRs), which interacts with the mRNA via three RNA recognition motif (RRM) field. Among them, two N-terminal domains (RRM1, RRM2) are responsible for binding U-rich RNA sequences, while the C-terminal domain (RRM3) is responsible for stabilizing the HuR-RNA complex. In addition, the hinge region which contains HuR nucleocytoplasmic shuttling sequence (HNS), is responsible for linking RRM2 and RRM3.[2] (See <a href="#ovhur2">(2) HuR and post-transcriptional regulation</a> for details)</p>
								<p style="text-align:center;">
									<span class="image fit"><img src="images/overview/hur01.jpg" alt="" ></img></span>
									<b>Figure. B-1. HuR protein and post-translational modification by cancer-related enzymes.</b>
								</p>
								<p style="color: black;">
									Reference: Uren, Philip J. et al. “Genomic Analyses of the RNA-Binding Protein Hu Antigen R (HuR) Identify a Complex Network of Target Genes and Novel Characteristics of Its Binding Sites.” The Journal of Biological Chemistry 286.43 (2011): 37063–37066. PMC. Web. 8 Oct. 2017.
								</p>
							</dd>
							<dt>2. It plays a key role in the inflammatory response. (See <a href="#ovhur3">(3) HuR and inflammation</a> for details)</dt>
							<dd>
								<p style="text-align:center;">
									<b>Table. B-1: HuR influence on inflammtory process.</b>
									<span class="image fit"><img src="images/overview/hur02.png" alt="" ></img></span>
								</p>
								<p style="color: black;">
									Reference: Uren, Philip J. et al. “Genomic Analyses of the RNA-Binding Protein Hu Antigen R (HuR) Identify a Complex Network of Target Genes and Novel Characteristics of Its Binding Sites.” The Journal of Biological Chemistry 286.43 (2011): 37063–37066. PMC. Web. 8 Oct. 2017.
								</p>
							</dd>
							<dt>3. It promotes the progress of cancer, including cell proliferation, drug resistance and angiogenesis. (See <a href="#ovhur4">(4) HuR and psoriasis</a> for details)</dt>
							<dd>
							</dd>
						</dl>
						<hr class="major">
						<h2 id="ovhur2">(2) HuR and post-transcriptional regulation</h2>
						<p style="text-align:center; color: black;">
							<span class="image fit"><img src="images/overview/hur03.png" alt="" ></img></span>
							<b>Figure. B-2. Hu protein domains and their functions.</b>
						</p>
						<p style="color: black;">
							Reference: Hinman, Melissa & Lou, Hua. (2008). Diverse molecular functions of Hu proteins. Cellular and molecular life sciences : CMLS. 65. 3168-81. 10.1007/s00018-008-8252-6.
						</p>
						<p>Since HuR associates with most of the intracellular reactions, such as mRNA nucleus-cytoplasm transportation and post-transcriptional modification, it is extremely crucial to physiological activities including cell metabolism and proliferation. Below are some examples of how HuR interacts with mRNA.
							<br>HuR can bind with AREs on 3’UTR  upon external or physiological stimulus. It will transport the mRNA from nucleus to cytoplasm, and further control the half life and translation of mRNA [4,5].</p>
						<p>RRM1 and RRM2 of HuR protein are involved in the recognition of mRNA, while RRM3 combines with poly(A) to maintain mRNA stability.
							<br>Previous study showed that HuR cannot interact with target mRNA when RRM3 was mutated, indicating that RRM3 plays the most important role when HuR acts out its functions [6,7,8].
						</p>
						<p>When HuR locates in the nucleus, it mainly combines with pre-mRNA to assist the formation of mature mRNA [2,9]. In addition, it’s related to spliceosome complex and controls the reporter genes splicing [10,11]. Also, it takes part in mRNA alternative splicing [12] through HuR phosphorylation.
							<br>When HuR is transported into the cytoplasm, it will stabilize mRNA by attaching to its 3’UTR, thus keeping the translation process going [13].
						</p>
						<p>Also, HuR can control the expression of mRNA isoforms by attaching to different polyadenylation sites on them [14,15].</p>
						<hr class="major">
						<h2 id="ovhur3">(3) HuR and inflammation</h2>
						<p>More than 90% of the cytokines’ and chemokines’ mRNAs contain AREs sequence, indicating the importance of post-transcriptional regulation in the immune responses [16]. The expression of HuR in fibroblasts, T cells and macrophages can stabilize mRNA of proinflammatory cytokines TNF-α and IL-6, and then induce inflammations [17-21].</p>
						<p>Furthermore, two proinflammatory mediators, iNOS and COX2, are also regulated by HuR. HuR enhances iNOS expression in muscle, hepatocytes and lung cancer and colorectal cancer cells via binding to 3'UTR of iNOS mRNA [22-24]. Elevated NO expression plays a critical role in inflammation, apoptosis, angiogenesis and cancer progress [25,26]. Meanwhile, HuR regulates the expression of COX2 in macrophages and glomeruli as well as the pathological growth of cancer cells such as breast cancer, ovarian cancer and colon cancer [27-29].</p>
						<p>In addition to inflammatory molecules, HuR can also inhibit the production of anti-inflammatory protein, thrombomodulin [30]. In view of the effect of HuR on inflammatory protein expression, several studies pointed out that HuR participates in different disease states. For instance, HuR promotes a large number of TNF-α performance in rheumatoid arthritis [31,32], and the expression of COX2 affects the erosion of cartilage in arthritis [33]. Besides, HuR can increase COX2 expression of intestinal epithelial cells in inflammatory bowel disease [34].</p>
						<p>In the process of chronic asthmatic disease, HuR increases the expression of TNF-α and GM-CSF [35] and dominates the inflammatory signaling transduction of the mediator protein or cytokines in the inflammatory diseases. Therefore, we can block the binding of HuR and mRNA as a potential treatment target.</p>
						<hr class="major">
						<h2 id="ovhur4">(4) HuR and psoriasis</h2>
						<p>Among macrophage-related inflammations, we chose the commonly-seen and easily-observed disease, psoriasis, as our model. The following is a quick look on the clinical symptoms and mechanism of psoriasis, providing an insight into the improvement of these symptoms through inhibiting HuR.</p>
						<p>Psoriasis is a long-term skin disease, which is characterized by the overgrowth of keratinocytes and vasculars along with immune cells infiltration in the skin lesions. It is associated with obesity and cardiovascular diseases. On the lesioned skin region, erythema  and scales can be significantly observed, which severely undermine patients’ quality of life [36].</p>
						<p>Similar to other chronic inflammatory diseases, such as rheumatoid arthritis and inflammatory bowel disease, Th1- and Th17-related cytokines, including TNF-α, Type I-IFN and IL-17, play a major role as pathogenic factor [37].
							<br>Immune cells, dendritic cells, T-cells, macrophages and leukocytes can cause infiltration in dermis [38], and epidermis (leukocytes only).When lots of leukocytes accumulate in the stratum corneum of epidermis, it is called microbe abscess[39].
						</p>
						<p>As keratinocytes in the epidermis, the main cells that make up our skin, keep proliferating and differentiating, they will normally differentiate into a sort of karyocytes (cells without nucleus) and attach to each other through junction complex to form a physical barrier [40] that separates our body from the outside world.
							<br>However, at the lesioned sites, keratinocytes will undergo abnormal overgrowth and differentiation, which leads to the continual existence of their cell nuclei and the soaring amount of themselves, since infiltrated immune cells will not only promote inflammations at the lesioned site through cytokines but also secrete growth factors including KGF, EGF, and VEGF. On the other hand, keratinocytes will produce cytokines, which trigger inflammatory, cytokines and chemokines such as CCL2 and CXCL12, which attract more immune cells to the lesioned sites [41].
						</p>
						<p>A large number of HuR proteins are detected in the cytoplasm of the psoriasis-keratinocytes. The HuR proteins contribute to the high expression of cytokine IL-20, and thus have an impact on the development of psoriasis [42]. In addition, according to a previous study, defective NF-κB (IκB) kinase 2 (IKK2) produced in the keratinocytes could possibly induce psoriasis [43]. However with a lack of the macrophages-produced cytokines, the induced psoriasis will not be able to maintain any longer. Further on the CD18 hypo mouse model, a study found that TNF-α produced by macrophages is the definitive factor to enable the maintenance of psoriasis [44]. Therefore, it is suggested that the inhibition of the HuR proteins in the keratinocytes and macrophages could be an effective strategy to alleviate the symptoms of psoriasis.</p>
						<h2>(5) HuR and cancer</h2>
						<p>Previous studies revealed that overexpressed HuR and abnormal nuclear-cytoplasmic ratio are highly correlated with several cancers. [45-48]
							<br>One of the studies proposed that owing to the character of HuR enhancing the levels of proteins which promotes cell proliferation, elevates cell survival rate, boosts local angiogenesis, assists the cancer cell evading from immune recognition, and facilitates cancer cell invasion and metastasis, HuR exerts a tumorigenic function by enabling these cancer phenotypes. [49]
						</p>
						<p>The biopsy samples of human small-cell lung carcinoma, esophageal squamous cell carcinoma, primary renal cell carcinoma, upper urinary tract urothelial carcinoma and bladder cancer show that the accumulation of HuR in cytoplasm has positive correlation with grading of biopsy, metastasis, degree of cancer cell invasion and survival rate [50-53]. Moreover, in cases of oral squamous cell carcinoma, the expression number of HuRs is positively correlated with oncogenes and proto-oncogenes, which severely affects the prognosis and survival rate of patients [54,55]. HuR stabilizes the mRNAs of oncogenes and related growth factors in cytoplasm as a means of cancer deterioration. As a consequence, we can predict cancer progress by observing the expression of HuRs accumulated in cytoplasm. Besides, some studies have pointed out that, once the anti-apoptotic Bcl-2s produced by highly-expressed HuRs are stabilized, glioma would be allowed to become more resistant to chemotherapeutics like Topotecan, Cisplatin and Etoposide [56].
							<br>The same research also indicates that breast cancer cells have higher expression of HuR, which can stabilize the drug-resistant proteins and activate cell signaling pathways like MAPK, JNK to reduce the efficacy of Tamoxifen (a chemotherapeutic) [57] .
							<br>As a result, if we could inhibit HuR activity in patients receiving chemotherapy, the therapeutic effect would be considerably improved.
						</p>
						<hr class="major">
						<h2>(6) HuR inhibitors as potential anti-inflammatory and anti-cancer drugs</h2>
						<p>Since HuR plays a pivotal role in the development of chronic inflammation, cancer, as well as other diseases, recently it has become a research subject that many scientists are highly interested in. The reasons why HuR is an ideal drug target are shown in Table. B-2 below, including the HuR involvement in the etiology of many different diseases, such as cancer, inflammation, and neurological disease, HuR’s well characterized structure, and its ubiquitously expression, and so on.</p>
						<p style="text-align:center; color: black;">
							<b>Table. B-2: The advantages of choosing HuR as drug target.</b>
							<span class="image fit"><img src="images/overview/hur04.png" alt="" ></img></span>
						</p>
						<p style="color: black;">
							Reference: Zucal C, D'Agostino V, Loffredo R, Mantelli B, NatthakanThongon, Lal P, Latorre E, Provenzani A Curr Drug Targets. 2015;16(5):499-515.
						</p>
						<p>However, there are only a few studies on the development of HuR inhibitors. So far, globally related researches show the following kinds of inhibitors, some possessing other biological activities (Table. B-3).</p>
						<p style="text-align:center;">
							<b>Table. B-3: Existing HuR inhibitors.</b>
							<span class="image fit"><img src="images/overview/hur05.png" alt="" ></img></span>
						</p>
						<p style="color: black;">Reference:
							<ol style="list-style-type:upper-roman; color:black;">
								<li>Cheng, Y. C.; Liou, J. P.; Kuo, C. C.; Lai, W. Y.; Shih, K. H.; Chang, C. Y.; Pan, W. Y.; Tseng, J. T.; Chang, J. Y. Mol Cancer Ther 2013, 12, (9), 1919-1919.</li>
								<li>Lee, J. Y.; Chung, T. W.; Choi, H. J.; Lee, C. H.; Eun, J. S.; Han, Y. T.; Choi, J. Y.; Kim, S. Y.; Han, C. W.; Jeong, H. S.; Ha, K. T. Biochem Bioph Res Co 2014, 447, (2), 371-377.</li>
								<li>Sun, L. X.; Zhang, S.; Jiang, Z. Z.; Huang, X.; Wang, T.; Huang, X.; Li, H.; Zhang, L. Y. Biochem Bioph Res Co 2011, 416, (1-2), 99-105.</li>
								<li>Chae, M. J.; Sung, H. Y.; Kim, E. H.; Lee, M.; Kwak, H.; Chae, C. H.; Kim, S.; Park, W. Y. Exp Mol Med 2009, 41, (11), 824-831.</li>
								<li>Kwak, H.; Jeong, K. C.; Chae, M. J.; Kim, S. Y.; Park, W. Y.  Bmb Rep 2009, 42, (1), 41-46.</li>
								<li>Meisner, N. C.; Hintersteiner, M.; Mueller, K.; Bauer, R.; Seifert, J. M.; Naegeli, H. U.; Ottl, J.; Oberer, L.; Guenat, C.; Moss, S.; Harrer, N.; Woisetschlaeger, M.; Buehler, C.; Uhl, V.; Auer, M. Nature Chemical Biology 2007, 3, (8), 508-515.</li>
								<li>Bolognani, F.; Gallani, A. I.; Sokol, L.; Baskin, D. S.; Meisner-Kober, N. J Neurooncol 2012, 106, (3), 531-42.</li>
								<li>Zhu, Z.; Zhao, Y.; Li, J.; Tao, L.; Shi, P.; Wei, Z.; Sheng, X.; Shen, D.; Liu, Z.; Zhou, L.; Tian, C.; Fan, F.; Shen, C.; Zhu, P.; Wang, A.; Chen, W.; Zhao, Q.; Lu, Y. Mol Carcinog 2016, 55, (10), 1399-410.</li>
								<li>Shin, D. S.; Kim, H. N.; Shin, K. D.; Yoon, Y. J.; Kim, S. J.; Han, D. C.; Kwon, B. M. Cancer Research 2009, 69, (1), 193-202.</li>
							</ol>
						</p>
						<p>Even these small-molecular compounds are proven to have biological activity to inhibit HuR function, their <em>in vivo</em> therapeutic efficacy is rarely confirmed. RRM1 and RRM2 in HuR family, via modeling analysis by computer, are found to share a highly-conserved similar sequence. It has been shown that the competitive combination of chemically synthesized small fragment peptides to these two sites is found to stabilize expression of NOVA-1 and VEGF mRNA [58, 59]. This indicates that small-molecular drugs with binding affinity can be discovered through computer simulation.</p>
						<hr class="major">
						<p><b>Reference:</b>
							<blockquote>
								<ol>
									<li>Dassi, E.; Zuccotti, P.; Leo, S.; Provenzani, A.; Assfalg, M.; D'Onofrio, M.; Riva, P.; Quattrone, A. Nucleic Acids Res 2013, 41, (5), 3201-16.</li>
									<li>Mukherjee, N.; Corcoran, D. L.; Nusbaum, J. D.; Reid, D. W.; Georgiev, S.; Hafner, M.; Ascano, M.; Tuschl, T.; Ohler, U.; Keene, J. D. Mol Cell 2011, 43, (3), 327-339.</li>
									<li>Peng, S. S.; Chen, C. Y.; Xu, N.; Shyu, A. B. Embo J 1998, 17, (12), 3461-70.</li>
									<li>Peng, S. S. Y.; Chen, C. Y. A.; Xu, N. H.; Shyu, A. B. Embo J 1998, 17, (12), 3461-3470. </li>
									<li>de Silanes, I. L.; Zhan, M.; Lal, A.; Yang, X. L.; Gorospe, M. P Natl Acad Sci USA 2004, 101, (9), 2987-2992. 		</li>
									<li>Hinman, M. N.; Lou, H. Cell Mol Life Sci 2008, 65, (20), 3168-3181.</li>
									<li>Brennan, C. M.; Steitz, J. A. Cell Mol Life Sci 2001, 58, (2), 266-277. </li>
									<li>Ma, W. J.; Chung, S.; Furneaux, H. Nucleic Acids Research 1997, 25, (18), 3564-3569.</li>
									<li>Lebedeva, S.; Jens, M.; Theil, K.; Schwanhausser, B.; Selbach, M.; Landthaler, M.; Rajewsky, N. Mol</li>
									<li>Chen,Y.I.G.;Moore,R.E.;Ge,H.Y.;Young,M.K.;Lee,T.D.;Stevens,S.W.NucleicAcidsResearch 2007, 35, (12), 3928-3944. </li>
									<li>Wang, H. W.; Molfenter, J.; Zhu, H.; Lou, H. Nucleic Acids Research 2010, 38, (11), 3760-3770. </li>
									<li>Akaike, Y.; Masuda, K.; Kuwano, Y.; Nishida, K.; Kajita, K.; Kurokawa, K.; Satake, Y.; Shoda, K.; Imoto, I.; Rokutan, K. Mol Cell Biol 2014, 34, (15), 2857-2873.</li>
									<li>Dutertre, M.; Chakrama, F. Z.; Combe, E.; Desmet, F. O.; Mortada, H.; Espinoza, M. P.; Gratadou, L.; Auboeuf, D. Nat Commun 2014, 5. </li>
									<li>Naipauer, J.; Gattelli, A.; Degese, M. S.; Slomiansky, V.; Wertheimer, E.; Lamarre, J.; Castilla, L.; Abba, M.; Kordon, E. C.; Coso, O. A. Biochem J 2013, 454, 345-357.</li>
									<li>Zhu, H.; Zhou, H. L.; Hasman, R. A.; Lou, H. Journal of Biological Chemistry 2007, 282, (4), 2203- 2210.</li>
									<li>Anderson, P. Nature Reviews Immunology 2010, 10, (1), 24-35.</li>
									<li>Nabors, L. B.; Gillespie, G. Y.; Harkins, L.; King, P. H. Cancer Research 2001, 61, (5), 2154-2161.</li>
									<li>Dixon, D. A.; Tolley, N. D.; King, P. H.; Nabors, L. B.; McIntyre, T. M.; Zimmerman, G. A.; Prescott, S. M. J Clin Invest 2001, 108, (11), 1657-1665.</li>
									<li>Sung, S. C.; Kim, K.; Lee, K. A.; Choi, K. H.; Kim, S. M.; Son, Y. H.; Moon, Y. S.; Eo, S. K.; Rhim, B. Y. J Vasc Res 2009, 46, (1), 36-44. </li>
									<li>Wang, J. G.; Collinge, M.; Ramgolam, V.; Ayalon, O.; Fan, X. H. C.; Pardi, R.; Bender, J. R. Journal of Immunology 2006, 176, (4), 2105-2113. </li>
									<li>Zhou, H.; Jarujaron, S.; Gurley, E. C.; Chen, L.; Ding, H.; Studer, E.; Pandak, W. M., Jr.; Hu, W.; Zou,	T.; Wang, J. Y.; Hylemon, P. B. Atherosclerosis 2007, 195, (1), e134-43. 					</li>
									<li>Di Marco, S.; Mazroui, R.; Dallaire, P.; Chittur, S.; Tenenbaum, S. A.; Radzioch, D.; Marette, A.; Gallouzi, I. E. Mol Cell Biol 2005, 25, (15), 6533-45. </li>
									<li>Linker,K.;Pautz,A.;Fechir,M.;Hubrich,T.;Greeve,J.;Kleinert,H.NucleicAcidsRes2005,33,(15), 4813-27.</li>
									<li>Matsui, K.; Nishizawa, M.; Ozaki, T.; Kimura, T.; Hashimoto, I.; Yamada, M.; Kaibori, M.; Kamiyama, Y.; Ito, S.; Okumura, T. Hepatology 2008, 47, (2), 686-97.</li>
									<li>Srikantan, S.; Gorospe, M. Front Biosci (Landmark Ed) 2012, 17, 189-205.</li>
									<li>Kusaka, G.; Uno, K.; Iijima, K.; Shimosegawa, T. World J Gastrointest Pathophysiol 2016, 7, (1), 131- 7.</li>
									<li>Dixon, D. A.; Tolley, N. D.; King, P. H.; Nabors, L. B.; McIntyre, T. M.; Zimmerman, G. A.; Prescott, S. M. J Clin Invest 2001, 108, (11), 1657-65.</li>
									<li>Sengupta, S.; Jang, B. C.; Wu, M. T.; Paik, J. H.; Furneaux, H.; Hla, T. J Biol Chem 2003, 278, (27), 25227-33.</li>
									<li>Cok, S. J.; Acton, S. J.; Morrison, A. R. Journal of Biological Chemistry 2003, 278, (38), 36157-36162.</li>
									<li>Yeh, C. H.; Hung, L. Y.; Hsu, C.; Le, S. Y.; Lee, P. T.; Liao, W.  L.; Lin, Y. T.; Chang, W. C.; Tseng, J.  T. Mol Biol Cell 2008, 19, (9), 3812-3822.</li>
									<li>Sugihara, M.; Tsutsumi, A.; Suzuki, E.; Wakamatsu, E.; Suzuki, T.; Ogishima, H.; Hayashi, T.; Chino, Y.; Ishii, W.; Mamura, M.; Goto, D.; Matsumoto, I.; Ito, S.; Sumida, T. Arthritis Rheum 2007, 56, (7), 2160-2169.</li>
									<li>Suzuki, E.; Tsutsumi, A.; Sugihara, M.; Mamura, M.; Goto, D.; Matsumoto, I.; Ito, S.; Ikeda, K.; Ochiai, N.; Sato, Y.; Sumida, T. Int J Mol Med 2006, 18, (2), 273-278.</li>
									<li>Nieminen, R.; Vuolteenaho, K.; Riutta, A.; Kankaanranta, H.; van der Kraan, P. M.; Moilanen, T.; Moilanen, E. European Journal of Pharmacology 2008, 587, (1-3), 309-316.</li>
									<li>Di Mari, J. F.; Saada, J. I.; Mifflin, R. C.; Valentich, J. D.; Powell, D. W.  Am J Physiol-Gastr L  2007, 293, (4), G719-G728.</li>
									<li>Esnault, S.; Malter, J. S. Journal of Immunology 2003, 171, (12), 6780-6787.</li>
									<li>Correia, B.; Torres, T. Acta Biomed 2015, 86, (2), 121-9</li>
									<li>Reich, K. J Eur Acad Dermatol 2012, 26, 3-11.</li>
									<li>Griffiths, C. E. M.; Barker, J. N. W. N. Lancet 2007, 370, (9583), 263-271.</li>
									<li>Christophers, E.; Metzler, G.; Rocken, M. Brit J Dermatol 2014, 170, (1), 59-65.</li>
									<li>Bowcock, A. M.; Krueger, J. G. Nature Reviews Immunology 2005, 5, (9), 699-711.</li>
									<li>Lowes, M. A.; Bowcock, A. M.; Krueger, J. G. Nature 2007, 445, (7130), 866-873.    </li>
									<li>Garcin, G.; Guiraud, I.; Lacroix, M.; Genthon, C.; Rialle, S.; Joujoux, J. M.; Meunier, L.; Lavabre- Bertrand, T.; Stoebner, P. E.; Le Gallic, L. J Invest Dermatol 2015, 135, (11), 2732-2741.</li>
									<li>Stratis, A.; Pasparakis, M.; Rupec, R. A.; Markur, D.; Hartmann, K.; Scharffetter-Kochanek, K.; Peters, T.; van Rooijen, N.; Krieg, T.; Haase, I. J Clin Invest 2006, 116, (8), 2094-104.</li>
									<li>Wang, H.; Peters, T.; Kess, D.; Sindrilaru, A.; Oreshkova, T.; Van Rooijen, N.; Stratis, A.; Renkl, A. C.; Sunderkotter, C.; Wlaschek, M.; Haase, I.; Scharffetter-Kochanek, K. J Clin Invest 2006, 116, (8), 2105- 14.</li>
									<li>Lopez de Silanes, I.; Fan, J.; Yang, X.; Zonderman, A. B.; Potapova, O.; Pizer, E. S.; Gorospe, M.Oncogene 2003, 22, (46), 7146-54.</li>
									<li>Denkert, C.; Koch, I.; von Keyserlingk, N.; Noske, A.; Niesporek, S.; Dietel, M.; Weichert, W. Mod Pathol 2006, 19, (9), 1261-9.</li>
									<li>Yi, X.; Zhou, Y.; Zheng, W.; Chambers, S. K. Aust N Z J Obstet Gynaecol 2009, 49, (1), 93-8.</li>
									<li>Lim, S. J.; Kim, H. J.; Kim, J. Y.; Park, K.; Lee, C. M. Int J Gynecol Pathol 2007, 26, (3), 229-234.</li>
									<li>Abdelmohsen, Kotb, and Myriam Gorospe. “Post-Transcriptional Regulation of Cancer Traits by HuR.” Wiley interdisciplinary reviews. RNA 1.2 (2010): 10.1002/wrna.4. PMC. Web. 8 Oct. 2017.</li>
									<li>Wang, J.; Wang, B.; Bi, J.; Zhang, C. Med Oncol 2011, 28 Suppl 1, S577-85.</li>
									<li>Zhang, C.; Xue, G. L.; Bi, J. W.; Geng, M.; Chu, H. L.; Guan, Y. P.; Wang, J.; Wang, B. C. Tumor  Biol 2014, 35, (1), 73-80.</li>
									<li>Liang, P. I.; Li, W. M.; Wang, Y.  H.; Wu, T. F.; Wu, W. R.; Liao, A. C.; Shen, K. H.; Wei, Y. C.;  Hsing, C. H.; Shiue, Y. L.; Huang, H. Y.; Hsu, H. P.; Chen, L. T.; Lin, C. Y.; Tai, C.; Lin, C. M.; Li, C. F. Bmc Cancer 2012, 12.</li>
									<li>Miyata, Y.; Watanabe, S.; Sagara, Y.; Mitsunari, K.; Matsuo, T.; Ohba, K.; Sakai, H. Plos One 2013, 8, (3).</li>
									<li>Cha, J. D.; Li, S.; Cha, I. H. Head Neck-J Sci Spec 2011, 33, (5), 627-637.</li>
									<li>Cha, J. D.; Kim, H. K.; Cha, I. H. Head Neck-J Sci Spec 2014, 36, (8), 1168-1175.</li>
									<li>Filippova, N.; Yang, X. H.; Wang, Y. M.; Gillespie, G. Y.; Langford, C.; King, P. H.; Wheeler, C.; Nabors, L. B. Mol Cancer Res 2011, 9, (5), 648-659.</li>
									<li>Hostetter, C.; Licata, L. A.; Witkiewicz, A.; Costantino, C. L.; Yeo, C. J.; Brody, J. R.; Keen, J. C. Cancer Biol Ther 2008, 7, (9), 1498-1508.</li>
									<li>Rossi, D.; Amadio, M.; Carnevale Baraglia, A.; Azzolina, O.; Ratti, A.; Govoni, S.; Pascale, A.; Collina, S. J Med Chem 2009, 52, (16), 5017-9.</li>
									<li>Amadio, M.; Pascale, A.; Govoni, S.; Laurini, E.; Pricl, S.; Gaggeri, R.; Rossi, D.; Collina, S. Chem Biol Drug Des 2013, 81, (6), 707-14.</li>
								</ol>
							</blockquote>
						</p>
					</section>
				</div>
			</div>

			<!-- Sidebar -->
			<div id="sidebar">
				<div class="inner">
					<nav id="menu">
						<section>
							<img width="83%" style="margin: auto; display: block;" src="images/logo_bar.png"></img>
						</section>
						<header class="major" style="margin-bottom:2em;">
						</header>
						<ul>
							<li><a href="/">Homepage</a></li>
							<li>
								<span class="opener">Overview</span>
								<ul>
									<li><a href="overview_idea">Idea</a></li>
									<li><a href="overview_hur">HuR Introduction</a></li>
									<li><a href="overview_js1">JS-1 Design</a></li>
								</ul>
							</li>
							<li>
								<span class="opener">Experiment</span>
								<ul>
									<li><a href="experiment_purpose">Purpose</a></li>
									<li><a href="experiment_design">Design</a></li>
									<li><a href="experiment_labbook">LabBook</a></li>
									<li><a href="experiment_invitro">In Vitro</a></li>
									<li><a href="experiment_invivo">In Vivo</a></li>
									<li><a href="experiment_drugcarrier">Drug Carrier</a></li>
								</ul>
							</li>
							<li><a href="simulation">Simulation</a></li>
							<li>
								<span class="opener">Discussion</span>
								<ul>
									<li><a href="discussion#dis1">Biosafety</a></li>
									<li><a href="discussion#dis2">Cell Targeting and Drug Carrier</a></li>
									<li><a href="discussion#dis3">Other Diseases</a></li>
									<li><a href="discussion#dis4">Hu Protein Family</a></li>
									<li><a href="discussion#dis5">Outlook</a></li>
								</ul>
							</li>
							<li>
								<span class="opener">About</span>
								<ul>
									<li><a href="about-team">About our team</a></li>
									<li><a href="about-sponsors">About sponsors</a></li>
								</ul>
							</li>
						</ul>
					</nav>
					<section>
						<header class="major">
							<h2>Contact us</h2>
						</header>
						<ul class="contact">
							<li class="fa-facebook-official"><a href="https://www.facebook.com/NCKUBiomod/">成大Biomod分子設計研究會</a></li>
						</ul>
					</section>
					<footer id="footer">
						<p class="copyright">&copy; Team JSbond. All rights reserved. <br>Special thanks to theme designer: HTML5 UP.</p>
					</footer>
				</div>
			</div>
		</div>

				<!-- Scripts -->
				<script src="assets/js/jquery.min.js"></script>
				<script src="assets/js/skel.min.js"></script>
				<script src="assets/js/util.js"></script>
				<script src="assets/js/main.js"></script>
				<script src="assets/js/jquery.magnific-popup.min.js"></script>
				<script src="assets/js/dms.js"></script>
	</body>
</html>
